build/download-saturn-l2.js

```diff
@@ -67,12 +67,12 @@ async function main () {
           await pipeline(res.body, gunzip(), tar.extract(outDir))
         } else {
           // Darwin needs to be a zip for notarization
-          await mkdir(path.join(outDir, 'l2node-darwin-x64'))
+          await mkdir(path.join(outDir, 'l2node-darwin-x64'), { recursive: true })
           const parser = unzip.Parse()
           await Promise.all([
             (async () => {
               while (true) {
-                const entry = /** @type UnzipStreamEntry */ (await once(parser, 'entry'))
+                const [entry] = /** @type {[UnzipStreamEntry]} */ (await once(parser, 'entry'))
```

Comment:
thank you so much for fixing! and weird that it worked before that, I must not have tested properly